src/test/java/org/jenkinsci/plugins/saml/LiveTest.java

```diff
@@ -174,7 +174,8 @@ private static SamlSecurityRealm configureBasicSettings(IdpMetadataConfiguration
         // TODO use @DataBoundSetter wherever possible and load defaults from DescriptorImpl
         File samlKey = new File(Jenkins.get().getRootDir(), "saml-key.jks");
         FileUtils.copyURLToFile(LiveTest.class.getResource("LiveTest/saml-key.jks"), samlKey);
-        SamlEncryptionData samlEncryptionData = new SamlEncryptionData(samlKey.getAbsolutePath(), Secret.fromString("changeit"), Secret.fromString("changeit"), null, false);
+        SamlEncryptionData samlEncryptionData = new SamlEncryptionData(samlKey.getAbsolutePath(), Secret.fromString(
+                "changeit"), Secret.fromString("changeit"), null, false, false);
```

Comment:
Why not
```suggestion
                "changeit"), Secret.fromString("changeit"), null, false, true);
```

since it was working with this before?